index.html

```diff
@@ -0,0 +1,344 @@
+			                </div>
+			                <div class="tabs__content-text-block">
+				            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod 
+					        bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra  
+					        justo commodo. Proin sodales pulvinar tempor. Cum sociis natoque 
+					        penatibus et magnis dis parturient montes.
+					       <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod 
+					        bibendum laoreet.</p>
+			                </div>
+							<div>
+								<button class="tabs__content-buttom">Download</button>
+							</div>
+						    </div>
+						    <div class="tabs__content-img">
+							    <img class="tabs__img" src="./png/NewYork.png" alt="Statue of Liberty in New York"> <!--чето не то с размером-->
+						   </div>
+                    </div>
+		    </div>
+		</section>
+		<section class="awesome">
+			<div class="awesome__title-wrapper">
+				<h2 class="awesome__title title">WHY THIS IS AWESOME</h2>
+			</div>
+				<div class="awesome__line line"></div>
+				<p class="awesome__subtitle subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+				<div class="awesome__container">
+					<div class="awesome__block">
+						<div class="awesome__icon-wrapper">
+							<img class="awesome__icon" src="./png/Vector 3.png" alt="">
+						</div>
+						<h4 class="awesome__block-title">Thoughtful Design</h4>
+						<p class="awesome__block-subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing 
+							elit. Aenean euismod bibendum laoreet. Proin gravida  dolor sit amet lacus accumsan et viverra.</p>
+					</div>
+					<div class="awesome__block">
+						<div class="awesome__icon-wrapper">
+							<img class="awesome__icon" src="./png/Vector.png" alt="">
+						</div>
+						<h4 class="awesome__block-title">Thoughtful Design</h4>
+						<p class="awesome__block-subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing 
+							elit. Aenean euismod bibendum laoreet. Proin gravida  dolor sit amet lacus accumsan et viverra.</p>
+					</div>
+					<div class="awesome__block">
+						<div class="awesome__icon-wrapper">
+							<img class="awesome__icon" src="./png/Vector 2.png" alt="">
+						</div>
+						<h4 class="awesome__block-title">Thoughtful Design</h4>
+						<p class="awesome__block-subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing 
+							elit. Aenean euismod bibendum laoreet. Proin gravida  dolor sit amet lacus accumsan et viverra.</p>
+					</div>
+					
+				</div>
+				</p>
+		</section>
+		<section class="pricing">
+			<div class="pricing__title-wrapper">
+				<h2 class="pricing__title title">PRICING OPTIONS</h2>
+			</div>
+				<div class="pricing__line line"></div> 
+				<p class="pricing__subtitle subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+			<div class="pricing__price">
+				<div class="pricing__price-box">
+				<div class="pricing__price-basic">
```

Comment:
You don't need different classnames for each pricing block. They have completely same styles.